tests/Form/Admin/CreateNewUserTypeTest.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace App\Tests\Form\Admin;
+
+use App\Form\Type\Admin\UpdateUserType;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Form\Extension\Core\Type\EmailType;
+use Symfony\Component\Form\Extension\Core\Type\NumberType;
+use Symfony\Component\Form\Extension\Core\Type\PasswordType;
+use Symfony\Component\Form\Extension\Core\Type\SubmitType;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\Form\FormBuilderInterface;
+
+class CreateNewUserTypeTest extends TestCase
+{
+    private UpdateUserType $updateUserType;
+
+    protected function setUp(): void
+    {
+        $this->updateUserType = new UpdateUserType();
+    }
+
+    private function testBuildForm(): void
+    {
+        $form = $this->createMock(FormBuilderInterface::class)
```

Comment:
it looks like you are trying to test the Symfony functionality. this test doesn't really make sense